include/openthread/instance.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
  * @note This number versions both OpenThread platform and user APIs.
  *
  */
-#define OPENTHREAD_API_VERSION (203)
+#define OPENTHREAD_API_VERSION (204)
```

Comment:
@samkumar , with #7583 merged, can you help rebase this PR and increment `OPENTHREAD_API_VERSION` again? Thanks.

Reply:
Never mind, I think it is fine to keep the same version number since I'll merge these back-to-back.